ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -2929,10 +2929,10 @@ private void setStatsPropAndAlterPartition(boolean resetStatistics, Table tbl,
   }
 
   private void setStatsPropAndAlterPartitions(boolean resetStatistics, Table tbl,
-                                             List<Partition> partitions,
+                                              List<Partition> partitions,
```

Comment:
I believe this change is unintentional. Can you remove this change?

Rest LGTM.

Reply:
Hi @rbalamohan, this was misaligned and I thought I should make it aligned if I do a fix in this method. Should I remove it?